### DISCUSSION TITLE
janssen installation facing issue

### DISCUSSION THREAD

**Discussion by thimmeshasm** [JanssenProject/jans#10499] [Show and tell]
![image](https://github.com/user-attachments/assets/88d17c2b-1767-4e60-9128-98f51a6b941c)
how to find the context path

**Comment by thimmeshasm** [comment]
docker-jans-all-in-one aftet installing it not browsing anything

**Comment by nynymike** [comment]
unclear how to reproduce your error. What instructions did you follow? Did you check https://docs.jans.io/v1.2.0/janssen-server/install/docker-install/quick-start/

To check your installation, it's best to run the TUI, as the above instructions suggest.
